flow/src/main/scala/org/alephium/flow/io/PendingTxStorage.scala

```diff
@@ -21,9 +21,12 @@ import org.rocksdb.{ReadOptions, WriteOptions}
 import org.alephium.flow.model.PersistedTxId
 import org.alephium.io._
 import org.alephium.protocol.model.TransactionTemplate
-import org.alephium.storage.KeyValueStorage
-import org.alephium.storage.rocksdb.{RocksDBKeyValueCompanion, RocksDBKeyValueStorage, RocksDBSource}
-import org.alephium.storage.rocksdb.RocksDBSource.ColumnFamily
+import org.alephium.storage.{ColumnFamily, KeyValueStorage}
+import org.alephium.storage.rocksdb.{
+  RocksDBKeyValueCompanion,
+  RocksDBKeyValueStorage,
+  RocksDBSource
+}
```

Comment:
multi-line imports are discouraged

Reply:
This looks like an older commit. Should be single in the latest.